engine/extend/legacy_plugins.md

```diff
@@ -64,6 +64,7 @@ Plugin
 [Local Persist Plugin](https://github.com/CWSpear/local-persist)                    | A volume plugin that extends the default `local` driver's functionality by allowing you specify a mountpoint anywhere on the host, which enables the files to *always persist*, even if the volume is removed via `docker volume rm`.
 [NetApp Plugin](https://github.com/NetApp/netappdvp) (nDVP)                         | A volume plugin that provides direct integration with the Docker ecosystem for the NetApp storage portfolio. The nDVP package supports the provisioning and management of storage resources from the storage platform to Docker hosts, with a robust framework for adding additional platforms in the future.
 [Netshare plugin](https://github.com/ContainX/docker-volume-netshare)                 | A volume plugin that provides volume management for NFS 3/4, AWS EFS and CIFS file systems.
+[Nimble Storage plugin](https://infosight.nimblestorage.com/InfoSight/app#software/Integration+Kits/Linux+Toolkit+%28NLT%29) | A volume plug-in that integrates seamlessly with high-performance Nimble Storage Unified Flash Fabric arrays. The plug-in abstracts all array volume capabilities (such as dedupe, encryption, performance policies, volume placement etc) to the Docker administrator to allow true self-provisioning of secure multi-tenant volumes and clones.
```

Comment:
This reads a bit like marketing copy. Can you reduce the description of Nimble Storage and just link to a (non-login-required) page which talks about its capabilities? The other entries here seem a little (though not much) less jargony.